_posts/en/newsletters/2019-04-09-newsletter.md

```diff
@@ -0,0 +1,309 @@
+  to skipping verification of signatures on all transactions included
+  in blocks before the assumed valid block.  The program still keeps
+  track of the bitcoin ownership changes produced by each transaction in
+  a index called the Unspent Transaction Output (UTXO) set.  Although
+  reviewing each historic ownership change still takes time, simply
+  skipping signature checking reduces initial sync time by about 80%
+  according to [tests][0.14 tests].  Gregory Maxwell, who [implemented
+  the assumed valid feature][Bitcoin Core #9484], has argued that,
+  "Because the validity of a chain history is a simple objective fact,
+  it is [easy] to review this setting."
+
+    This week James O'Beirne started a [thread][assumeutxo thread] on
+    the Bitcoin-Dev mailing list about taking a hash of the UTXO set at
+    a particular block, having multiple well-known contributors verify
+    they get the same hash, and then having freshly installed Bitcoin
+    Core nodes default to using that hash to download the exact same
+    UTXO set.  This would allow a newly-started node to skip not only
+    signatures but all block chain data before the assumed valid block,
+    perhaps reducing the bandwidth and time requirements to start a node
+    today by 95% or more (and certainly more as the block chain
+    continues to grow).  This is an old idea, is currently implemented
+    in some software that uses Bitcoin Core, and is part of the
+    motivation for research into other techniques such as [fast
+    updatable UTXO commitments][] and [automatic levelDB backups][].
+
+    Discussion mainly revolved around whether or not this is a good
+    idea.  Arguments in favor of it include it making starting a new
+    node much easier and that it doesn't seem to change the trust model
+    for anyone who already trusts the peer review of their development
+    team or who is willing to disable the default option to have their
+    node review the chainstate for validity itself.  Arguments against
+    it include a fear that fast initial syncs with an assumed valid UTXO
+    set would disguise the fact that block size increases make complete
+    initial syncing from scratch much more expensive; if block sizes
+    increased too much, it could become impossible for anyone of modest
+    means to ever trustlessly verify Bitcoin's UTXO state, forcing new
+    users to trust existing users.
+
+## Bech32 sending support
+
+*Week 4 of 24.  Until the second anniversary of the segwit soft
+fork lock-in on 24 August 2019, the Optech Newsletter will contain this
+weekly section that provides information to help developers and
+organizations implement bech32 sending support---the ability to pay
+native segwit addresses.  This [doesn't require implementing
+segwit][bech32 easy] yourself, but it does allow the people you pay to
+access all of segwit's multiple benefits.*
+
+<div class="hide-on-web" markdown="1">
```

Comment:
It's a little bit better that we don't include it on the live website.  I'm no SEO expert, but it's my understanding than search engines don't like `display: none` content because it gets indexed by their crawlers and so may show up in search results but the user won't see it if they view the page.  Scammers thus use it to inflate their keyword mentions or something.  So I'm removing it.  Here's the HTML for reference (obviously, feel free to rephrase however you'd like):

```html
<p><strong>Note:</strong> we’re unable to correctly format multi-line code examples in
the email edition of the newsletter.  Please visit the <a href="/en/newsletters/2019/04/09/">web
edition</a> for better formatting.  We apologize for the
inconvenience.</p>
```